deluge/core/core.py

```diff
@@ -359,8 +359,8 @@ def do_remove_torrents():
             for torrent_id in torrent_ids:
                 try:
                     self.torrentmanager.remove(torrent_id, remove_data=remove_data, save_state=False)
-                except InvalidTorrentError as ex:
-                    errors.append((torrent_id, ex))
+                except InvalidTorrentError:
+                    errors.append(torrent_id)
```

Comment:
The problem is simply that exceptions can't be serialized by rencode. The tests pass because it's run locally without serializing. I would prefer a solution where the reason could be returned as part of the return value. We could return the exceptions getMessage value?